plugins/modules/net_tools/nmcli.py

```diff
@@ -1536,6 +1537,7 @@ def ip_conn_type(self):
             'team',
             'vlan',
             'wifi',
+            '802-11-wireless',
```

Comment:
Do these changes need a mention in the changelog?

Reply:
done